clients/cobol-lsp-vscode-extension/.vscode/settings.json

```diff
@@ -13,12 +13,15 @@
  */
 // Place your settings in this file to overwrite default and user settings.
 {
-	"files.exclude": {
-		"out": false // set this to true to hide the "out" folder with the compiled JS files
-	},
-	"search.exclude": {
-		"out": true // set this to false to include "out" folder in search results
-	},
+    "files.exclude": {
+        "out": false
+        // set this to true to hide the "out" folder with the compiled JS files
+    },
+    "search.exclude": {
+        "out": true
+        // set this to false to include "out" folder in search results
+    },
     "typescript.tsdk": "./node_modules/typescript/lib",
-    "vsicons.presets.angular": false, // we want to use the TS server from our node_modules folder to control its version
+    "vsicons.presets.angular": false
+    // we want to use the TS server from our node_modules folder to control its version
```

Comment:
Do we keep comments inside json-elements?

Reply:
Good question, is modified because I made a mess removing that file and rollback it back, btw I had the feeling that is a kind of leftover that I could get rid of it

Reply:
In fact, the specification of JSON doesn't allow comments inside the structure. But I don't think it's critical in this case